### ISSUE TITLE
window operation test

### ISSUE THREAD

**Issue by xmnlab** [ibis-project/ibis#1770]
hey everyone! I have stated the implementation of window operation for MapD/OmniSci and I have a question related to tests.

is this parameter working? 

```
param(
            lambda t: t.float_col.rank(),
            lambda t: t,
            id='rank',
            # marks=pytest.mark.xfail,
),
```

locally it is raising the follow error:

```
test_window.py:148: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/mnt/sda1/storage/miniconda/envs/ibis/lib/python3.6/site-packages/pandas/core/frame.py:3551: in assign
    data[k] = com.apply_if_callable(v, data)
/mnt/sda1/storage/miniconda/envs/ibis/lib/python3.6/site-packages/pandas/core/frame.py:3370: in __setitem__
    self._set_item(key, value)
/mnt/sda1/storage/miniconda/envs/ibis/lib/python3.6/site-packages/pandas/core/frame.py:3445: in _set_item
    value = self._sanitize_column(key, value)
/mnt/sda1/storage/miniconda/envs/ibis/lib/python3.6/site-packages/pandas/core/frame.py:3630: in _sanitize_column
    value = sanitize_index(value, self.index, copy=False)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

data = <pandas.core.groupby.generic.DataFrameGroupBy object at 0x7f0b1073cac8>, index = RangeIndex(start=0, stop=7300, step=1), copy = False

    def sanitize_index(data, index, copy=False):
        """
        Sanitize an index type to return an ndarray of the underlying, pass
        through a non-Index.
        """
    
        if index is None:
            return data
    
        if len(data) != len(index):
>           raise ValueError('Length of values does not match length of index')
E           ValueError: Length of values does not match length of index

```

**Comment by cpcloud** [comment]
This was closed by #1778 and #1779

**Comment by xmnlab** [comment]
thanks @cpcloud !
